test/integration/utils/enroll.js

```diff
@@ -23,6 +23,16 @@ async function enrollMember({ mr, tk, tc }, members, kycAuthSigner, options = {}
     await tk.connect(member).approve(tc.address, ethers.constants.MaxUint256);
     await tk.transfer(member.address, initialTokens);
   }
+  const membershipApprovalData0 = await signMembershipApproval({
```

Comment:
The top level function is called enrollMember and enrolls a set of members (could be better called enrollMembers but nvm for now)

Why does it run an extra routine here to enroll one of the contracts a member?

This logic should not exist here from the point of view of what this function should do

This logic should exist wherever this registration of cl is necessary but not in this commonly used function for enrolling members.